parm/wave/wave_gfs_pruned.buoys

```diff
@@ -0,0 +1,614 @@
+$
```

Comment:
Can this be a different name and be in a different PR.  This is unrelated to the reorg of COM.

Reply:
This is the previous `wave_gfs.buoys` file. I didn't want to lose it when I replaced it with the version from fix. If we back out that change, this one will go away too (but boundary point jobs would fail again because there are none in this file).